Node/core/lib/bots/ChatConnector.js

```diff
@@ -671,8 +674,8 @@ var ChatConnector = (function () {
                                     callback(null, response, body);
                                 }
                                 else {
-                                    var txt = options.method + " to '" + options.url + "' failed: [" + response.statusCode + "] " + response.statusMessage;
```

Comment:
this is just cleanup to make it consistent with the above code, which is the same